src/Aspire.Hosting/ApplicationModel/ResourceWithConnectionStringSurrogate.cs

```diff
@@ -15,4 +15,7 @@ internal sealed class ResourceWithConnectionStringSurrogate(IResource innerResou
     }
 
     public string? ConnectionStringEnvironmentVariable => environmentVariableName;
+
+    public ReferenceExpression ConnectionStringExpression =>
+        ReferenceExpression.Create($"");
```

Comment:
This seems like it isn't going to work. How is a "surrogate" supposed to work here?

Reply:
I missed this one.

Reply:
This "works" because we never write this value to the manifest. This is always written as a parameter.